cassandra/writer.go

```diff
@@ -147,11 +152,14 @@ func (w *Writer) worker() {
 			err := w.write(id, s.TimestampMS, s.Value)
 			if err != nil {
 				// send error back on payload's errch; don't block the worker
+				go ll.WithError(err).Error("write failed")
+
 				select {
 				case m.errch <- err:
 				default:
 				}
 			}
+			ll.WithFields(log.Fields{"sample": s.Value, "id": id}).Debug("sample written to cassandra")
```

Comment:
I don't like such frequent debug level logs. I don't see the value, and it's such a critical code path here that it seems like a waste of cycles. I don't think golang can compile out this code when running in >DEBUG level, so it's not free.

We're tracking the number of writes already via a prometheus metric.